Generics.md

```diff
@@ -73,21 +73,21 @@ Commit and distribute to CMR as done in the addition steps.
 ## Generic Document Pipeline API Endpoints
 
 The API endpoints are generated from a pre-configured list of document types. The type is singular in
-ingest and plural in search. The examples below use the "orderoption"/"orderoptions" type, with a
-native ID of "orderoption-1".
+ingest and plural in search. The examples below use the "order-option"/"order-options" type, with a
+native ID of "order-option-1".
 
 #### ingest
-	curl -v -XPOST -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/orderoption/orderoption-1?provider=PROV1" -d @orderoption-1.json
-	curl -v -XPUT -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/orderoption/orderoption-1?provider=PROV1" -d @orderoption-1.json
-	curl -v -XGET -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/orderoption/orderoption-1?provider=PROV1"
-	curl -v -XDELETE -H "$TOKEN" "https://cmr.earthdata.nasa.gov/ingest/orderoption/dataqualitysummary-1?provider=PROV1"
+	curl -v -XPOST -H "$TOKEN" -H "Content-Type:application/vnd.nasa.cmr.umm+json" "https://cmr.earthdata.nasa.gov/ingest/order-option/order-option-1?provider=PROV1" -d @order-option-1.json
```

Comment:
reading this today, I think the native id should look more different then the type

Reply:
I agree that it would be more readable if we made our example native-id's something that does not look like concept-type or Name at first glance.